docs/medical-api/more-info/webhooks.mdx

```diff
@@ -359,3 +359,47 @@ Example payload:
   ]
 }
 ```
+
+#### Bulk Document Download
```

Comment:
why is this a subsection of consolidated? should be `###`, no?

also, this let's call it what it is "Bulk Document Download URLs"